FE/pages/index.tsx

```diff
@@ -1,17 +1,25 @@
-import type { NextPage } from 'next';
+import type { GetServerSideProps, NextPage } from 'next';
 import Head from 'next/head';
 import Image from 'next/image';
 import tw from 'twin.macro';
 import Lottie from 'lottie-react-web';
 import animation from '../public/animation-lottie.json';
 import AuthModal from '../components/AuthModal';
 import { useState } from 'react';
+import ContributorList from '../components/contributors';
+import { IContributors } from '../interfaces';
 
-const Home: NextPage = () => {
+type Props = {
+  contributors: IContributors[];
+};
+
+const Home = ({ contributors }: Props) => {
```

Comment:
any specific reason to delete NextPage as type for Home component?

Reply:
It's only a type for the return, not that big of a deal if it's not included, I added it back anyway.
ref: https://stackoverflow.com/questions/70623841/usage-of-nextpage-type-in-next-js